app/events/consignment_inquiry_event.rb

```diff
@@ -13,6 +13,7 @@ def subject
   def properties
     {
       email: @object.email,
+      recipient_email: @object.recipient_email,
```

Comment:
**suggestion:** Could we add a comment to explain how `recipient_email` is different to `email`?

Reply:
👍 useful. Thanks. I just did.

Reply:
Thanks :)